### ISSUE TITLE
After i used it, I have some issues follow.

### ISSUE THREAD

**Issue by zeronlee** [dubox/jSearch#1]
- I searched with 'site:google.com', I hope it can be configured automatically.
- Add custom shortcut, eg: I usually use the space to see next screen, the result conflicted, I see the search input.
- Every morning, I watch top news by Baidu, Zhihu, Sf and so on, when I saw your soft, I thought of this usage, now, I hope I input nothing with entering, the result show headlines.
- I not used to drag the scroll bar, because I feel it very tired at a window.
![image](https://user-images.githubusercontent.com/6701474/53777059-03bb4f80-3f33-11e9-85a5-9c46c871b6e8.png)

In short, It is a very thoughtful software, hope is to get better.

**Comment by dubox** [comment]
Thank you for your suggestions, I will seriously consider about these. 
And it will be  implemented in subsequent versions which any of these suggestions if it's useful and doable.

**Comment by dubox** [comment]
The last three suggestions have been solved in v0.5.0. There is no plan for the first one. Thank you for your suggestions.
